back/config/locales/en.yml

```diff
@@ -134,6 +134,15 @@ en:
       attachments:
         title: Attachments
         description:
+      section1:
```

Comment:
I don't think it's necessary to introduce this, as it will always be overridden, and might cause unnecessary translation work.

Reply:
Removed title, although left description there as a blank placeholder to match the others

Reply:
I didn't realize there were already `description` keys with blank values in there. I don't see the value of those either, but since they're already there I think it's good indeed to keep things consistent.